addons/TBMod_main/configs/Cfg3DEN.hpp

```diff
@@ -16,53 +16,53 @@ class Cfg3DEN
-                        class TB_mission_extremMedic
-                        {
-                            displayName = "extremes Sanisystem";
-                            tooltip = "Arzt kann nur noch EHK verwenden usw. GENEMIGUNGSPFLICHTIG!";
-                            property = "TB_main_mission_extremMedic";
-                            control = "Checkbox";
-                            expression = "if (!is3DEN) then {TB_extremSanisystem = _value; publicVariable 'TB_extremSanisystem'};";
-                            defaultValue = "false";
-                        };
-                        class TB_mission_crashHelfer
-                        {
-                            displayName = "Crashhelfer";
-                            tooltip = "Wenn jemand die Mission verlässt, wird alles über ihn gespeichert.";
-                            property = "TB_mission_crashHelfer";
-                            control = "Checkbox";
-                            expression = "if (!is3DEN) then {TB_crashHelfer = _value; publicVariable 'TB_crashHelfer'};";
-                            defaultValue = "true";
-                        };
-                        class TB_mission_doorBreach
-                        {
-                            displayName = "DoorBreach";
-                            tooltip = "Schrotflinten SLUG Munition kann verschlossene Türen öffnen.";
-                            property = "TB_mission_doorBreach";
-                            control = "Checkbox";
-                            expression = "if (!is3DEN) then {TB_doorBreach = _value; publicVariable 'TB_doorBreach'};";
-                            defaultValue = "false";
-                        };
-                        class TB_mission_recoilSystem
-                        {
-                            displayName = "RecoilSystem Koeffizent";
-                            tooltip = "Die Stärke des dynamischen Rückstoßes. -1 = AUS";
-                            property = "TB_mission_recoilSystem";
-                            control = "Edit";
-                            expression = "if (!is3DEN) then {TB_recoilCoef = _value; publicVariable 'TB_recoilCoef'};";
-                            defaultValue = "1";
-                            validate = "number";
-                            typeName = "NUMBER";
-                        };
-                    };
-                };
+                    // class Attributes
+                    // {
+                        // class TB_mission_extremMedic
+                        // {
+                            // displayName = "extremes Sanisystem";
+                            // tooltip = "Arzt kann nur noch EHK verwenden usw. GENEMIGUNGSPFLICHTIG!";
+                            // property = "TB_main_mission_extremMedic";
+                            // control = "Checkbox";
+                            // expression = "if (!is3DEN) then {TB_extremSanisystem = _value; publicVariable 'TB_extremSanisystem'};";
```

Comment:
Warum fliegt das alles raus?

Reply:
- Extremes Sanisystem gibtsnicht mehr
- crashhelfer wird nur noch hidden verboten werden können in 2 Jahren noch nie jemand false gemacht
- doorbreach kann per Variable angemacht werden, wurde 2 mal nur con mir benutzt
- recoilsystem soll CBA setting werden